### PR TITLE
Add CPI & CSI package for OCI into the community-edition

### PR DESCRIPTION
**Pull request by HanFa**
## What this PR does / why we need it

Add the OCI CPI and CSI package to TCE, so that it can be consumed as a package in the management and workload clusters.

## Which issue(s) this PR fixes

Fixes: #

## Describe testing done for PR

ytt package unit test

### CODE REVIEW

File: addons/packages/oracle-cpi/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  name: oracle-cpi.community.tanzu.vmware.com
+spec:
+  displayName: "oracle-cpi"
+  longDescription: "Kubernetes Cloud Controller Manager implementation for Oracle Cloud Infrastucture"
+  shortDescription: "OCI Kubernetes Cloud Controller Manager"
+  providerName: VMware
+  maintainers:
+    - name: Naadir Jeewa
+    - name: Fang Han
```
Comment (randomvariable) [inline addons/packages/oracle-cpi/metadata.yaml:12]:
```suggestion
    - name: Naadir Jeewa <jeewan@vmware.com>
    - name: Fang Han <fhan@vmware.com>
```

File: addons/packages/oracle-cpi/1.22.1/bundle/config/provider-config.lib.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  #                    allows inbound traffic to the appropriate ports for kube
+  #                    proxy health port, node port ranges, and health check port ranges.
+  #                    E.g. 10.82.0.0/16 30000-32000.
+  #   "None":          Disables all security list management. Requires that the
+  #                    user has setup a rule that allows inbound traffic to the
+  #                    appropriate ports for kube proxy health port, node port
+  #                    ranges, and health check port ranges. E.g. 10.82.0.0/16 30000-32000.
+  #                    Additionally requires the user to mange rules to allow
+  #                    inbound traffic to load balancers.
+  securityListManagementMode: All
```
Comment (randomvariable) [inline addons/packages/oracle-cpi/1.22.1/bundle/config/provider-config.lib.txt:38]: This should be overridable

Comment (randomvariable) [reply addons/packages/oracle-cpi/1.22.1/bundle/config/provider-config.lib.txt:38]: Also, checking with Shyam, if the mode is all, we do also then have to set https://github.com/oracle/oci-cloud-controller-manager/blob/master/manifests/provider-config-example.yaml#L56-L58

Comment (HanFa) [reply addons/packages/oracle-cpi/1.22.1/bundle/config/provider-config.lib.txt:38]: i've kept that noted. let's have a separate PR to address this

File: addons/packages/oracle-cpi/1.22.1/bundle/.imgpkg/bundle.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: imgpkg.carvel.dev/v1alpha1
+kind: Bundle
+metadata:
+  name: oracle-cpi
+authors:
+- name: Naadir Jeewa
+  email: jeewan@vmware.com
```
Comment (randomvariable) [inline addons/packages/oracle-cpi/1.22.1/bundle/.imgpkg/bundle.yml:7]: @HanFa , oi, add yourself too!

Comment (HanFa) [reply addons/packages/oracle-cpi/1.22.1/bundle/.imgpkg/bundle.yml:7]: added

**Pull request merged**